### ISSUE TITLE
Not install any extension

### ISSUE THREAD

**Issue by WiNE-iNEFF** [tachiyomiorg/tachiyomi-extensions#10125]
### Source information

Newbie, remanga, mangalib, henchan (all version)

### Steps to reproduce

1. Install from repo extension differrent extension
2. Open installer

### Expected behavior

When you click on install, "installation is not possible" is displayed

### Actual behavior

.

### Tachiyomi version

r4011

### Android version

8.1.1

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.12.3](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

**Comment by arkon** [comment]
Try downloading/installing from https://tachiyomi.org/extensions/

**Comment by WiNE-iNEFF** [comment]
This page dont loading

**Comment by arkon** [comment]
You can try using a VPN if content is blocked by your country/ISP/etc.

**Comment by WiNE-iNEFF** [comment]
> Попробуйте скачать / установить с https://tachiyomi.org/extensions/

this link does not work for me

**Comment by WiNE-iNEFF** [comment]
still not working. the page only loads halfway

**Comment by arkon** [comment]
You'll have to debug your issue yourself, since there's clearly something wrong with your particular device/environment.
